### ISSUE TITLE
[v2] Introspection query not returning custom scalars

### ISSUE THREAD

**Issue by hailwood** [jimkyndemeyer/js-graphql-intellij-plugin#173]
## Version and Environment Details
__Operation system__: 
Ubuntu 18.04

__IDE name and version__: 
PhpStorm 2018.2 EAP
Build #PS-182.3684.40, built on July 18, 2018
JRE: 1.8.0_152-release-1248-b8 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Linux 4.15.0-24-generic

__Plugin version__: 
2.0.0-alpha-3

## Expected Behaviour
Valid schema generated via introspection

## Actual Behaviour
Missing custom scalar types causing schema to not validate

## Steps to Reproduce / Link to Repo with Reproduction and Instructions
define a schema with a "Timestamp" custom scalar
Introspect the server using that schema.

Note the saved .graphql schema does not have `scalar Timestamp` meaning any types that use said scalar do not validate.

The introspection from v1 did successfully add these scalar types.

**Comment by brabeji** [comment]
I confirm this this is an issue in version `2.0.0-alpha-3`
```
WebStorm 2018.2
Build #WS-182.3684.70, built on July 20, 2018
JRE: 1.8.0_152-release-1248-b8 x86_64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
macOS 10.13.5
```

**Comment by daggerjames** [comment]
Manually edit schema.graphql generated would workaround this issue, but not in recommended ways

**Comment by jimkyndemeyer** [comment]
The workaround for now is to simply declare the custom scalars in a new .graphql file like you would for framework-specific directives.

**Comment by jimkyndemeyer** [comment]
Fixed in the upcoming 2.0.0-alpha-7.
